deno.json

```diff
@@ -1,5 +1,9 @@
 {
-  "workspace": ["./bin/lang-js", "./lib/ts-lib-deno"],
+  "workspace": [
+    "./bin/lang-js",
+    "./lib/ts-lib-deno",
+    "./bin/si-gen-cloud-control"
```

Comment:
I think this is going to cause issues with the firecracker rootfs build. I'll dig in a bit to verify it.

Reply:
I can't run deno locally without it :/

Reply:
https://github.com/systeminit/si/pull/5295

We'll want to merge this first and then rebase